src/helm/benchmark/contamination/export_scenario_text.py

```diff
@@ -0,0 +1,91 @@
+import json
+import os
+import argparse
+from typing import List, DefaultDict
+from collections import defaultdict
+from copy import deepcopy
+
+from helm.common.general import asdict_without_nones, ensure_directory_exists
+from helm.common.hierarchical_logger import hlog, htrack_block
+
+from helm.benchmark.scenarios.scenario import Scenario, create_scenario, Instance
+from helm.benchmark.presentation.run_entry import read_run_entries
+from helm.benchmark.run import run_entries_to_run_specs
+from helm.benchmark.runner import RunSpec
+from helm.benchmark.contamination.light_scenario import LightInstance, LightScenario
+
+
+def create_light_instance_from_helm_instance(instance: Instance) -> LightInstance:
+    """Create a LightInstance given a helm Instance. Only keep the text attributes."""
+    input_text: str = instance.input.text
+    reference_texts: List[str] = [reference.output.text for reference in instance.references]
+    return LightInstance(input=input_text, references=reference_texts)
+
+
+def get_light_scenarios_from_helm(
+    run_spec: RunSpec, scenario_download_path: str = "exported_scenarios"
+) -> List[LightScenario]:
+    """
+    Create a list of LightInstances given a helm run_spec. Only keep the text attributes. Note that one LightScenario
+    object is created for each split of the helm scenario for simplification.
+    """
+
+    scenario: Scenario = create_scenario(run_spec.scenario_spec)
+
+    ensure_directory_exists(scenario_download_path)
+    scenario.output_path = os.path.join(scenario_download_path, scenario.name)
+    ensure_directory_exists(scenario.output_path)
+
+    helm_scenario_spec = {"name": scenario.name}
+    helm_scenario_spec.update(run_spec.scenario_spec.args)
```

Comment:
the scenario spec should just be `run_spec.scenario_spec` (don't just take args out).  Or you take `run_spec.name` instead, not `scenario.name`